lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py

```diff
@@ -184,12 +195,13 @@ def cacheFileUsdHierarchyOptions(topForm):
         for label in _listEditedAsLabels:
             cmds.menuItem(label=label)
 
-    variantRb = cmds.radioButtonGrp('variantRadioBtn',
-                                    nrb=1,
-                                    changeCommand1=variantOrNewPrim,
-                                    label=getMayaUsdLibString('kTextDefineIn'),
-                                    l1=getMayaUsdLibString('kTextVariant'),
-                                    annotation=getMayaUsdLibString('kTextVariantToolTip'))
+    rl = mel.eval('createRowLayoutforMayaReference("' + widgetColumn + '", "usdCacheDefineInRow", 3)')
+    with mayaRefUtils.SetParentContext(rl):
+        cmds.text(label=getMayaUsdLibString('kTextDefineIn'))
+        cmds.checkBoxGrp('variantCheckBox',
```

Comment:
At runtime, the radio-buttons could not see each-other to form a radio group. I think the same issue arose in the add-ref UI used when adding a Maya reference as it uses check-boxes too instead of radio-buttons.

Reply:
Did you confirm the change with Natalia? If she wants radio buttons, then just each one in its own collection.

Reply:
I'll ask but I was under the impression that when using radio buttons, one in each group would always need to be selected so both would be on. That was the problem I had when I used individual radio buttons: because they no longer were in the same group both were selected!

Reply:
I've been able to put back radio buttons by using a trick of creating invisible ones to allow the visible ones to be unselected.